be/src/runtime/exec_env.cpp

```diff
@@ -154,6 +154,13 @@ Status ExecEnv::_init(const std::vector<StorePath>& store_paths) {
     _pipeline_prepare_pool =
             new PriorityThreadPool("pip_prepare", num_prepare_threads, config::pipeline_prepare_thread_pool_queue_size);
 
+    int num_sink_io_threads = config::pipeline_sink_io_thread_pool_thread_num;
+    if (num_sink_io_threads <= 0) {
+        num_sink_io_threads = std::thread::hardware_concurrency();
+    }
+    _pipeline_sink_io_pool =
+            new PriorityThreadPool("pip_sink_io", num_sink_io_threads, config::pipeline_sink_io_thread_pool_queue_size);
+
```

Comment:
should we check `pipeline_sink_io_thread_pool_queue_size` ?

Reply:
> should we check `pipeline_sink_io_thread_pool_queue_size` ?

I think it's better to check, I have added a check logic above this line